easybuild/easyconfigs/x/xtb/xtb-6.2.3-foss-2019a.eb

```diff
@@ -0,0 +1,41 @@
+easyblock = 'MesonNinja'
+
+name = 'xtb'
+version = '6.2.3'
+
+homepage = 'https://xtb-docs.readthedocs.io'
+description = """ xtb - An extended tight-binding semi-empirical program package. """
+
+toolchain = {'name': 'foss', 'version': '2019a'}
+
+source_urls = [GITHUB_LOWER_SOURCE]
+sources = [{'download_filename': 'v%(version)s.tar.gz', 'filename': SOURCE_TAR_GZ}]
+checksums = ['620dfb93aa07fd5144d25134613a5d7f4c1fa4fb5070c30768a6335c8c0b4187']
+
+github_account = 'grimme-lab'
+builddependencies = [
+    ('Meson', '0.51.2', '-Python-3.7.4', ('GCCcore', '8.3.0')),
+    ('Ninja', '1.9.0', '', ('GCCcore', '8.3.0')),
```

Comment:
@dplacencia You should not hardcode the toolchain here for these dependencies. This is also why the tests are failing: `foss/2019b` is based on `GCCcore/8.2.0`, while you're forcing the use of `GCCcore/8.3.0` here.

If you want to use Python 3.7.4, you should update to the toolchain `foss/2019b`:

```python
toolchain = {'name': 'foss', 'version': '2019b'}
...
dependencies = [
    ('Meson', '0.51.2', '-Python-3.7.4'),
    ('Ninja', '1.9.0'),
]
```

If you prefer sticking to `foss/2019a`, you'll need to adjust the line for `Meson`:

```python
('Meson', '0.50.0', '-Python-3.7.2'),